spec/round_spec.rb

```diff
@@ -0,0 +1,137 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+require 'rspec'
+
+
+
+RSpec.describe Round do
+  it 'is a deck' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+
+    expect(round).to be_instance_of(Round)
+
+  end
+
+  it 'has a attributes' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+
+
+    expect(round.deck).to eq(deck)
+    expect(round.turns).to eq([])
+  end
+
+  it 'current card' do
```

Comment:
This is fine, but it can be helpful to be more explicit in your test names about what exactly you're testing. For example, this test might be called `it 'has the first card in the deck as current_card' do`.